src/Sulu/Bundle/ContentBundle/Behat/BaseStructureContext.php

```diff
@@ -146,6 +146,10 @@ protected function createStructureTemplate($type, $name, $template)
     {
         $paths = $this->getContainer()->getParameter('sulu.content.structure.paths');
         $paths = array_filter($paths, function ($value) use ($type) {
+            if (true === $value['internal']) {
```

Comment:
Do not include internal paths as candidate places to install our new structure template.